tests/api_tests/cuda/test_cuda_add_documents.py

```diff
@@ -212,21 +216,12 @@ def test_add_documents_defaults_to_cuda(self):
 
         self.client.create_index(self.index_name_1, settings_dict=index_settings)
 
-        self.client.index(self.index_name_1).add_documents([{"_id": "explicit_cpu", "title": "blah"}], device="cpu",
-                                                           non_tensor_fields=[])
-        self.client.index(self.index_name_1).add_documents([{"_id": "explicit_cuda", "title": "blah"}], device="cuda",
-                                                           non_tensor_fields=[])
-        self.client.index(self.index_name_1).add_documents([{"_id": "default_device", "title": "blah"}],
-                                                           non_tensor_fields=[])
-
-        cpu_vec = self.client.index(self.index_name_1).get_document(document_id="explicit_cpu", expose_facets=True)['_tensor_facets'][0]["_embedding"]
-        cuda_vec = self.client.index(self.index_name_1).get_document(document_id="explicit_cuda", expose_facets=True)['_tensor_facets'][0]["_embedding"]
-        default_vec = self.client.index(self.index_name_1).get_document(document_id="default_device", expose_facets=True)['_tensor_facets'][0]["_embedding"]
+        self.client.index(self.index_name_1).add_documents([{"_id": "default_device", "title": "blah"}], non_tensor_fields=[])
```

Comment:
let's start using `tensor_fields` moving forward with tests, so we won't have a hard time changing it later on

Reply:
updated